manager/controlapi/common.go

```diff
@@ -84,3 +87,17 @@ func validateDriver(driver *api.Driver) error {
 
 	return nil
 }
+
+// combineErrors combines multi errors into a single one.
+func combineErrors(errs []error) error {
```

Comment:
I think this approach to combining errors is okay since we're just returning the error string from a GRPC handler. But if we ever expose these errors to Go code, it might make sense to have a special type like this:

```go
type MultiError struct {
        Errors []error
}

func (me MultiError) Error() string {
        // ... join the error strings together here and return the result
}
```

That way, combining errors doesn't lose the type information.